### ISSUE TITLE
cl.exe error?

### ISSUE THREAD

**Issue by HyeokSuLee** [blaind/ttf2mesh-rs#1]
Enviroments : windows 10 +  vscode.

I think it belongs to msbuild. ubt  i have no idea how to handle it. 
any suggestion?

```The following warnings were emitted during compilation:

warning: cl : ������ error D8021 : '/Wextra' ���� �μ��� �߸��Ǿ����ϴ�.

error: failed to run custom build command for `ttf2mesh-sys v0.0.1`
Caused by:
  process didn't exit successfully: `I:\Documents\temp\../rust_build\release\build\ttf2mesh-sys-9cec62ce6690e430\build-script-build` (exit code: 1)
  --- stderr
  Assertation failed!

  Program: C:\Strawberry\c\bin\patch.exe
  File: .\src\patch\2.5.9\patch-2.5.9-src\patch.c, Line 354

  Expression: hunk


  error occurred: Command "I:\\VisualStudioCommunity2019\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-O2" "-Brepro" "-W4" "-Wall" "-Wextra" "-pedantic" "-std=c99" "-D_POSIX_C_SOURCE=199309L" "-FoI:\\Documents\\temp\\../rust_build\\release\\build\\ttf2mesh-sys-fb4c08238f6fc698\\out\\ttf2mesh/ttf2mesh.o" "-c" "ttf2mesh/ttf2mesh.c" with args "cl.exe" did not execute successfully (status code exit code: 2).```

**Comment by blaind** [comment]
Seems like Windows compilation required quite a few changes (didn't try before). 

Created a new branch at https://github.com/blaind/ttf2mesh-rs/tree/fix-windows

Before merging, I'd like to check whether the Path to *const i8 (over FFI) boundary conversion is okay. Current implementation at https://github.com/blaind/ttf2mesh-rs/commit/37915033559fbe867d5652b42a45f3b0ba062f0b#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R59 - comments on that?

ttf2mesh.c seems to pass the pointer directly to `fopen`.

**Comment by HyeokSuLee** [comment]
Cl errors gone. But path conversion doesn't work. 
```
cargo run --example ttf2check FiraMono-Medium.ttf thistext

Loading font "FiraMono-Medium.ttf"...
 - font load failed: FileNotFound
error: process didn't exit successfully: `target\debug\examples\ttf2check.exe FiraMono-Medium.ttf thistext` (exit code: 255)
```

**Comment by HyeokSuLee** [comment]
last issue solved. moved ttf file to top of the directory and works.
I was confused with each examples command args. it would be better if it's more specific.

thank you for your fast support.

ps. i think the code below is enough and shorter.

```
#[cfg(windows)]
fn path_to_cstring<P: AsRef<Path>>(path: P) -> CString {
    CString::new(path.as_ref().as_os_str().to_str().unwrap()).unwrap()
}
```

**Comment by HyeokSuLee** [comment]
merge plz.

**Comment by blaind** [comment]
Fixed and released in 0.0.10
